test/helpers/common.bash

```
@@ -75,6 +88,39 @@ shiplog_cleanup_sandbox_repo() {
+    local tmpdir
+    tmpdir=$(mktemp -d)
+    if ! ssh-keygen -q -t ed25519 -N '' -f "$tmpdir/id_ed25519"; then
+      echo "ERROR: Failed to generate SSH key" >&2
+      rm -rf "$tmpdir"
+      return 1
+    fi
+    git config gpg.format ssh
+    git config user.signingkey "$tmpdir/id_ed25519"
+    # TODO: Track tmpdir for cleanup
+  else
+    export GNUPGHOME="$(mktemp -d)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)